packages/client/components/SuggestedActionCard.tsx

```diff
@@ -54,7 +54,7 @@ const FloatingSealIcon = styled('div')({
   width: 36,
   position: 'absolute',
   svg: {
-    fontSize: 36
+    fontSize: 20
```

Comment:
Was around here and saw this:

<img width="568" alt="Screenshot 2023-07-10 at 16 27 19" src="https://github.com/ParabolInc/parabol/assets/1879975/cf7484b7-f9c6-4ef5-bb7d-3c34f93b4a4d">

So a quick fix